packages/backend/src/plugins/app.ts

```diff
@@ -5,15 +5,21 @@ import { resolvePackagePath } from '@backstage/backend-common';
 import { resolve as resolvePath } from 'path';
 import fs from 'fs-extra';
 import rateLimit from 'express-rate-limit';
+import { completeFrontendSchemas } from '../schemas';
 
 export default async function createPlugin(
   env: PluginEnvironment,
+  dynamicPluginsSchemas: { value: any; path: string }[],
 ): Promise<Router> {
   const appPackageName = 'app';
 
   const appDistDir = resolvePackagePath(appPackageName, 'dist');
   const staticDir = resolvePath(appDistDir, 'static');
 
+  if (await fs.pathExists(appDistDir)) {
+    await completeFrontendSchemas(dynamicPluginsSchemas, appDistDir);
```

Comment:
Does this mean if the `dist` doesn't exist we don't get schemas?

`yarn start` in dev mode doesn't create `package/app/dist`, so the default experience now means we do not aggregate schema properly.

Once I create the `package/app/dist` I get those schemas generated appropriately but they still do not get applied. 

What else do I have to do to make this work with `yarn start` locally?

Reply:
I assume that in the local dev mode, some changes should be done in the janus cli start command, to make it so that the webpack Dev server can also get the frontend configs related to dynamic plugins. 

That seems to be a completely distinct area and use-case, and maybe we could keep that use-case for a distinct issue ?

Reply:
I created a new issue for the Dev Server case: https://github.com/janus-idp/backstage-showcase/issues/763